tests/src/test/scala/system/basic/WskBasicPythonTests.scala

```diff
@@ -119,4 +122,37 @@ class CLIPythonTests
                     activation.logs.get.mkString("\n") should { not include ("pythonaction.py") and not include ("flask") }
             }
     }
+
+    behavior of "Python runtime"
+
+    it should "invoke a zipped Python action with virtualenv package" in withAssetCleaner(wskprops) {
```

Comment:
will this work with python:3?
have you tried?
if it does, great - i suspect it will because the python action unit tests run for both runtimes, so if you have a clean build, it will.
if it does not, the documentation should reflect the restriction.